### PR TITLE
CI: move to GitHub actions

### PR DESCRIPTION
**Pull request by klauer**
## Description
* Remove Travis CI Configuration
* Enable GitHub Actions on pcdsutils

## Motivation and Context
Result of actions from my fork with this PR branch: https://github.com/klauer/pcdsutils/actions/runs/3878848661

I'm not sure we can get pcdshub/pcdsutils to show Actions checks until the repository contains workflows itself. (I'd be happy to be proved wrong here, though)

### CODE REVIEW

File: .github/workflows/standard.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+on:
+  push:
+  pull_request:
+  release:
+    types:
+      - created
+
+jobs:
+  standard:
+    uses: klauer/pcds-ci-helpers/.github/workflows/python-standard.yml@enh_gha
```
Comment (klauer) [inline .github/workflows/standard.yml:12]: TODO: change workflow URL

**Review by tangkong — APPROVED** [review]
Results speak for themselves, looks good to me. 

This might be better addressed in the cookiecutter repo, but are we fully moving away from the `run_tests.py` file in favor of just running `pytest`?  If so we should change the documentation on the cookiecutter to reflect that.  (I don't think I saw that being changed, but also I don't see `run_tests.py` in there)

**Comment by klauer** [comment]
> This might be better addressed in the cookiecutter repo, but are we fully moving away from the `run_tests.py` file in favor of just running `pytest`? If so we should change the documentation on the cookiecutter to reflect that. (I don't think I saw that being changed, but also I don't see `run_tests.py` in there)

I'm of the opinion that `run_tests.py` should go away, as I think it's unnecessary boilerplate. We're a team (of course!), though, so we should agree on it first. 

I think the existence of the cookiecutter (whether used often or not) is a great place to show off all of our agreed-upon standards in a concrete way that can be replicated elsewhere (and propagated to existing repositories, when feasible)

**Comment by tangkong** [comment]
I've personally never used `run_tests.py`, and would be fine seeing it go 👍

File: run_tests.py [hunk 2]
```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env python
```
Comment (ZLLentz) [inline run_tests.py:1]: Good call on removing these run_tests files, it's good timing to do it

**Review by ZLLentz — APPROVED** [review]
looks good, just have to redirect the workflow url

File: .github/workflows/standard.yml (hunk 1 above)
Comment (klauer) [reply .github/workflows/standard.yml:12]:
```suggestion
    uses: pcdshub/pcds-ci-helpers/.github/workflows/python-standard.yml
```

**Comment by klauer** [comment]
Merging this and following up with a test PR to verify that actions are really working here (they are on my fork's [actions](https://github.com/klauer/pcdsutils/actions/runs/3888254812), as above)

**Pull request merged**